base/applications/regedit/lang/pt-PT.rc

```diff
@@ -103,35 +103,35 @@ BEGIN
         BEGIN
             MENUITEM "&Chave", ID_EDIT_NEW_KEY
             MENUITEM SEPARATOR
-            MENUITEM "Valor da &cadeia", ID_EDIT_NEW_STRINGVALUE
-            MENUITEM "Valor &Binário", ID_EDIT_NEW_BINARYVALUE
+            MENUITEM "&Valor da cadeia", ID_EDIT_NEW_STRINGVALUE
+            MENUITEM "Valor &binário", ID_EDIT_NEW_BINARYVALUE
             MENUITEM "Valor &DWORD", ID_EDIT_NEW_DWORDVALUE
             MENUITEM "Valor &Multi-cadeia", ID_EDIT_NEW_MULTISTRINGVALUE
-            MENUITEM "Valor c&adeia expansível", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
+            MENUITEM "Valor de cadeia e&xpansível", ID_EDIT_NEW_EXPANDABLESTRINGVALUE
```

Comment:
This section/submenu does exist 3 times in the file. I would recommend to use the same capitalization and the same accelerators for all 3 locations. And not use different ones for each.